pom.xml

```diff
@@ -1,334 +1,254 @@
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+  </build>
 
-    <dependencies>
-        <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>4.12</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.cxf</groupId>
-            <artifactId>cxf-rt-rs-client</artifactId>
-            <version>3.1.11</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.glassfish</groupId>
-            <artifactId>javax.json</artifactId>
-            <version>1.0.4</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.eclipse.microprofile</groupId>
-            <artifactId>microprofile</artifactId>
-            <version>1.3</version>
-            <scope>provided</scope>
-            <type>pom</type>
-        </dependency>
-        <dependency>
-            <groupId>com.ibm.websphere.appserver.api</groupId>
-            <artifactId>com.ibm.websphere.appserver.api.jaxrs20</artifactId>
-            <version>1.0.18</version>
-            <scope>provided</scope>
-        </dependency>
-        <dependency>
-            <groupId>com.ibm.websphere.appserver.api</groupId>
-            <artifactId>com.ibm.websphere.appserver.api.json</artifactId>
-            <version>1.0.18</version>
-            <scope>provided</scope>
-        </dependency>
-        <dependency>
-            <groupId>javax.ws.rs</groupId>
-            <artifactId>javax.ws.rs-api</artifactId>
-            <version>2.0.1</version>
-            <scope>provided</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.cxf</groupId>
-            <artifactId>cxf-rt-rs-extension-providers</artifactId>
-            <version>3.1.11</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>javax.servlet</groupId>
-            <artifactId>javax.servlet-api</artifactId>
-            <version>3.1.0</version>
-            <scope>provided</scope>
-        </dependency>
-
-    </dependencies>
-
-    <build>
+  <profiles>
+    <!-- mvn.cmd for windows systems -->
+    <profile>
+      <id>windowsExtension</id>
+      <activation>
+        <os><family>Windows</family></os>
+      </activation>
+      <properties>
+        <mvn.extension>.cmd</mvn.extension>
+      </properties>
+    </profile>
+    <!-- just mvn for other systems -->
+    <profile>
+      <id>nonWindowsExtension</id>
+      <activation>
+        <os><family>!Windows</family></os>
+      </activation>
+      <properties>
+        <mvn.extension></mvn.extension>
+      </properties>
+    </profile>
+    <!-- Open Liberty Profile -->
+    <profile>
+      <id>open-liberty</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
```

Comment:
I don't think there should be a default profile that chooses a particular runtime.

Any profile for a given runtime should have to be explicitly chosen